{{cookiecutter.project_slug}}/common/test/test_sample.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from django.urls import reverse, resolve
+
+from common import views
+
+
+class TestClass:
+
+    @pytest.mark.parametrize(
+        "url_name, url, view_class",
+        [
+            ("index", "/", views.IndexView),
+        ],
+    )
+    def test_urls(self, url_name, url, view_class):
+        resolver = resolve(url)
+        assert resolver.view_name == url_name
+        assert resolver.func.view_class == view_class
```

Comment:
This definitely handles the use case of giving people a sample to follow, but it would be more ideal to have a sample that 1. uses the database, and 2. tests something that is code we've written and not django code

Maybe we can create a new view that uses `LoginRequiredMixin` and test access that requires the use of factories and making database calls?